csrc/expr_evaluator.cpp

```diff
@@ -58,12 +58,6 @@ void ExpressionEvaluator::bind_(
     return;
   }
   TORCH_CHECK(!value->isConstScalar(), "Tried to bind to a constant value");
-  TORCH_CHECK(
```

Comment:
This is removed. If we already know an intermediate result, we should be able to just bind it, instead of having to start from the leaf and recompute. This change is required because the smem tvs are intermediates.